browsers/firefox_android.json

```diff
@@ -37,6 +72,21 @@
           "release_notes": "https://developer.mozilla.org/Firefox/Releases/10",
           "status": "retired"
         },
+        "11": {
+          "release_date": "2012-03-13",
+          "release_notes": "https://developer.mozilla.org/Firefox/Releases/11",
```

Comment:
Should maybe be [`https://www.mozilla.org/firefox/android/11.0beta/releasenotes/`](https://www.mozilla.org/firefox/android/11.0beta/releasenotes/).